### PR TITLE
feature/GE-636: Generated project, shifted settings, added pinned req…

### PR DESCRIPTION
**Pull request by AltusBarry**
…uirements, added apps, 2fa urls giving issues atm

### CODE REVIEW

File: project/settings_base.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/1.11/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = '_n(_w(3!i4-p((jz8(o0fb*_r5fb5t!qh1g@m9%4vryx5lale='
```
Comment (cobusc) [inline project/settings_base.py]: I presume we will read this from the environment when running on containers?

**Comment by cobusc** [comment]
:+1:

File: project/settings_base.py (hunk 1 above)
Comment (AltusBarry) [reply project/settings_base.py]: Ah yes, that one should be updated. Still a bit used to server local settings.

Comment (AltusBarry) [reply project/settings_base.py]: Base setting left as is, settings.py has env override.

**Pull request closed without merging**